test/cache_test.rb

```diff
@@ -144,6 +149,28 @@ class InheritedRoleSerializer < RoleSerializer
       @blog_serializer     = BlogSerializer.new(@blog)
     end
 
+    def test_expiring_of_cache_at_update_of_record
+      if ARModels::Author.respond_to?(:cache_versioning)
+        ARModels::Author.cache_versioning = true
+      end
+
+      author = ARModels::Author.create(name: 'Foo')
+      author_json = AuthorSerializerWithCache.new(author).as_json
+
+      assert_equal 'Foo', author_json[:name]
+
+      author.update_attributes(name: 'Bar')
```

Comment:
Is the reason for the db operation here to break the cache?

btw, should be `update` not `update_attributes`, and better to do `update!` to abort the test if a validation fails (not that it is likely, but still..)

Reply:
`Is the reason for the db operation here to break the cache?` yes updating an attribute should invalidate the cache :) this is what's broken currently for rails apps that have `cache_versioning` setted to `true`